MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDelegate.java

```diff
@@ -95,7 +95,7 @@ public float getElevation() {
      * @see View#setElevation(float)
      */
     public void setElevation(float elevation) {
-        boolean needsWrap = elevation > mElevation;
+        boolean needsWrap = elevation != mElevation;
```

Comment:
This change was done, because if we don't re-wrap when elevation decreases till 0, view's padding is being messed and the view "grows".